static/app/components/core/slider/index.chonk.tsx

```diff
@@ -0,0 +1,294 @@
+
+      &::-webkit-slider-runnable-track {
+        cursor: not-allowed;
+      }
+
+      &::-webkit-slider-thumb {
+        cursor: not-allowed;
+        border-bottom-width: 1px;
+      }
+    }
+
+    &::before {
+      content: '';
+      position: absolute;
+      inset: 0;
+      margin: auto 0;
+      min-width: calc(${p => p.theme.radius.micro} * 6);
+      width: var(--p, 50%);
+      height: 4px;
+      background: ${p => p.theme.colors.chonk.blue300};
+      border: 1px solid ${p => p.theme.colors.chonk.blue300};
+      border-radius: ${p => p.theme.radius.micro};
+    }
+
+    /* Chrome styling */
+    &::-webkit-slider-runnable-track {
+      width: 100%;
+      height: 4px;
+      background: ${p => p.theme.colors.surface100};
+      border: 1px solid ${p => p.theme.colors.surface100};
+      border-radius: ${p => p.theme.radius.micro};
+    }
+
+    &::-webkit-slider-thumb {
+      appearance: none;
+      width: 16px;
+      height: 16px;
+      background: ${p => p.theme.colors.white};
+      border: 1px solid ${p => p.theme.colors.chonk.blue100};
+      border-bottom: 2px solid ${p => p.theme.colors.chonk.blue100};
+      border-radius: ${p => p.theme.radius.sm};
+      transform: translateY(-7px);
+      z-index: 10;
+    }
+
+    /* Firefox styling */
+    &::-moz-range-track {
+      width: 100%;
+      height: 4px;
+      background: ${p => p.theme.colors.surface100};
+      border: 1px solid ${p => p.theme.colors.surface100};
+      border-radius: ${p => p.theme.radius.micro};
+    }
+
+    &::-moz-range-thumb {
+      appearance: none;
+      width: 16px;
+      height: 16px;
+      background: ${p => p.theme.colors.white};
+      border: 1px solid ${p => p.theme.colors.chonk.blue100};
+      border-bottom: 2px solid ${p => p.theme.colors.chonk.blue100};
+      border-radius: ${p => p.theme.radius.sm};
+      transform: translateY(-7px);
+      z-index: 1;
+    }
+`;
+
+const SliderOutput = chonkStyled('output')`
+  --tx: clamp(-50%, calc(-50% + var(--p, 0)), 50%);
+    --ty: var(--label-ty);
+    --o: var(--label-opacity);
+
+    /* disable interactions */
+    pointer-events: none;
+    user-select: none;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-size: ${p => p.theme.fontSizeSmall};
+    position: absolute;
+    height: 24px;
+    width: calc(100% - 16px);
+    margin-inline: auto;
+    left: 0;
+    right: 0;
+    bottom: 20px;
+    text-align: center;
+
+    opacity: var(--o);
+    transform: translate(var(--tx), var(--ty));
+    transition:
+      100ms opacity cubic-bezier(0.23, 1, 0.32, 1),
+      50ms transform cubic-bezier(0.39, 0.575, 0.565, 1);
+`;
+
+const SliderLabel = chonkStyled('span')`
+  font-size: inherit;
+  display: block;
+  min-width: calc(3ch + ${space(0.5)});
+  padding-inline: ${space(0.5)};
+  width: min-content;
+  text-align: center;
+  background: ${p => p.theme.colors.chonk.blue300};
+  border: 1px solid ${p => p.theme.colors.chonk.blue100};
```

Comment:
should we really use raw colors instead of semantic tokens?

Reply:
Actually, we don't seem to have the semantic tokens used in the designs in the codebase yet and this matches what the other chonk components do. Going to punt on this for now.